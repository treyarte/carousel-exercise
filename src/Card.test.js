import React from 'react';
import { render } from '@testing-library/react';
import Card from './Card';

it('should render a card', () => {
  render(<Card />);
});

it('should match the card snapshot', () => {
  const { asFragment } = render(<Card />);
  expect(asFragment).toMatchSnapshot();
});
