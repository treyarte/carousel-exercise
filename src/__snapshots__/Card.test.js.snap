// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the card snapshot 1`] = `[Function]`;
